### PR TITLE
New addon - Longer WIWO

### LINKED ISSUE #1521: 55 more chars in WIWO
**Describe the new addon**

Employs W_L's long wiwo hack. 

**Link to the new addon**

http://longwiwo.blogspot.com/

### PR DESCRIPTION
**Pull request by RedGuy12**
added new addon to resolve #1521

### CODE REVIEW

File: addons/longer-wiwo/userscript.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export default async function ({ msg }) {
+  const status = $('textarea[name="status"]');
+  const left = $($("#status-chars-left")[0].parentElement);
+
+  // remove scratch's event listeners
+  status.off("input");
+  status.off("focusin");
+  status.off("focusout");
```
Comment (WorldLanguages) [inline addons/longer-wiwo/userscript.js:8]: This seems to work, but if you're removing the event listeners, why does it still upload the updated WIWO when its changed? What am I missing? I want to know what's going on lol

File: addons/longer-wiwo/userscript.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+export default async function ({ msg }) {
+  const status = $('textarea[name="status"]');
+  const left = $($("#status-chars-left")[0].parentElement);
```
Comment (WorldLanguages) [inline addons/longer-wiwo/userscript.js:3]: `$("#status-chars-left").parent()` ?

File: addons/longer-wiwo/userscript.js (hunk 1 above)
Comment (WorldLanguages) [reply addons/longer-wiwo/userscript.js:8]: Also, I noticed that when the WIWO does update, Scratch adds a green modal to show it saved, but for some reason that modal is empty (has no text). Any ideas why?

Comment (RedGuy12) [reply addons/longer-wiwo/userscript.js:8]: 1: that event listener is on a different element, `div#status`
2: idk, i saw that before i even started working on this

File: addons/longer-wiwo/userscript.js (hunk 2 above)
Comment (RedGuy12) [reply addons/longer-wiwo/userscript.js]: ok

File: addons/longer-wiwo/userscript.js (hunk 1 above)
Comment (mxmou) [reply addons/longer-wiwo/userscript.js:8]: > Also, I noticed that when the WIWO does update, Scratch adds a green modal to show it saved, but for some reason that modal is empty (has no text). Any ideas why?

This is a Scratch bug (also happens without Scratch Addons). It's supposed to say "Changes saved".

File: addons/longer-wiwo/userscript.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+export default async function ({ msg }) {
+  const status = $('textarea[name="status"]');
+  const left = $($("#status-chars-left").parent()[0]);
```
Comment (WorldLanguages) [inline addons/longer-wiwo/userscript.js:3]: Huh, asking again, why not simply `const left = $("#status-chars-left").parent();`? Not sure why you're getting the real DOM element object to then convert it to a jQuery object again, seems unnecessary?

Comment (RedGuy12) [reply addons/longer-wiwo/userscript.js:3]: *facepalm* good point, didn't see that

Comment (WorldLanguages) [reply addons/longer-wiwo/userscript.js]: lol

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**